net-mgmt/pfSense-pkg-arpwatch/files/usr/local/pkg/arpwatch.inc

```diff
@@ -0,0 +1,239 @@
+<?php
+/*
+ * arpwatch.inc
+ *
+ * Copyright (c) 2018 Julien Le Goff <julego@gmail.com>
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+define('ARPWATCH_LOCAL_DIR', '/usr/local/arpwatch');
+define('ARPWATCH_ETHERCODES_URL', 'http://linuxnet.ca/ieee/oui/ethercodes.dat');
+define('ARPWATCH_SENDMAIL_PATH', '/usr/sbin/sendmail');
+define('ARPWATCH_SENDMAIL_PROXY', '/usr/local/arpwatch/sendmail_proxy.php');
+
+function arpwatch_install_command() {
+	global $config;
+
+	conf_mount_rw();
```

Comment:
We don't plan to bring new packages to 2.3.x, so all calls to `conf_mount_rw()` and `conf_mount_ro` should be removed